app/static/admin/js/pages/analytics/constants.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+define(function() {
+  return {
+    CREATED_AT_FORMATTED: "created_at_formatted",
```

Comment:
I wouldn't create a constants file for these kind of things. Perhaps you should just declare them at the top of each file or put them in the constructor of object. ie) `Teacher.constructor.type = "prof"` and you'd use `Teacher.constructor.type === type` to check whatever